src/ui/Toggle/ToggleContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ChangeEvent, useCallback, useState } from 'react';
+
+import {
+  ToggleContext,
+  ToggleContextInterface,
+  TOGGLE_DEFAULT_VALUE as Dvalue,
+} from './ToggleContext';
+
+export interface ToggleContainerProps extends ToggleContextInterface {
+  children?: React.ReactElement;
+}
+
+// Props Explanation https://github.com/aaronshaf/react-toggle#props
+export function ToggleContainer({
+  checked = Dvalue.checked, // null
+  defaultChecked = Dvalue.defaultChecked,
+  disabled = Dvalue.disabled,
+  onChange = Dvalue.onChange,
+  onFocus = Dvalue.onFocus,
+  onBlur = Dvalue.onBlur,
+  children = null,
+}: ToggleContainerProps): React.ReactElement {
+  const [isChecked, setChecked] = useState(defaultChecked || false);
+  const useOnChangeCallback = useCallback((e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
nit; 
```suggestion
  const handleChange = useCallback((e: ChangeEvent<HTMLInputElement>) => {
```
this kind of shorten name \wo `use` prefix would be fine here